server/integration/testsuite/src/test/java/org/infinispan/server/test/util/security/SslConfigurationHelper.java

```diff
@@ -0,0 +1,22 @@
+package org.infinispan.server.test.util.security;
+
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.commons.util.SslContextFactory;
+
+import javax.net.ssl.SSLContext;
+
+public class SslConfigurationHelper {
```

Comment:
SASL and SSL are something different but the base configuration is the same. Let me rename this file into `SecurityConfigurationHelper` and put those two things together.